src/app/Http/Requests/SapReports/ShowReportsRequest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Requests\SapReports;
+
+use App\Http\Requests\Base\DateRequest;
+
+/**
+ * A request to show SAP reports filtered by the date they were uploaded.
+ * 
+ * Fields: from, to.
+ */
+class ShowReportsRequest extends DateRequest
```

Comment:
to isté ako pri ShowOrExportOperations

Reply:
podobná odpoveď ako ShowOrExportOperationsRequest